content-security-policy/connect-src/connect-src-json-import-allowed.sub.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>connect-src-json-import-allowed</title>
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="connect-src 'self'; script-src http://{{host}}:{{ports[http][0]}}/resources/testharness.js http://{{host}}:{{ports[http][0]}}/resources/testharnessreport.js 'unsafe-inline';"
```

Comment:
I updated this test so that `script-src` does not include `self` and thus the test will fail if a platform mistakenly uses `script-src` for JSON (or CSS) modules.